### ISSUE TITLE
Repo Seems Abandoned, a new fork repo established with new release build 0.4.11 

### ISSUE THREAD

**Issue by stickoking** [1kc/razer-macos#861]
I forked this repo to add support for Blackwidow v4 75% and created a release I will be maintaining that repo for now, as this one seems abandoned below is the repos

https://github.com/stickoking/razer-macos
https://github.com/stickoking/librazermacos

Release https://github.com/stickoking/razer-macos/releases/tag/v0.4.11

**Comment by il-source** [comment]
does this work for anyone else? i have the white blackwidow v4 75% and tried downloading this. no luck so far…

**Comment by vanyasem** [comment]
@stickoking enable the "Issues" section on your fork (it's disabled by default on forks)
<img width="500" alt="Screenshot 2024-02-04 at 20 29 32" src="https://github.com/1kc/razer-macos/assets/13350458/e076eb92-f99c-4467-bcc4-3a23bd7bb271">

https://github.com/stickoking/razer-macos/settings

**Comment by stickoking** [comment]
I have enabled issues on forks @vanyasem sorry for that

**Comment by stickoking** [comment]
> does this work for anyone else? i have the white blackwidow v4 75% and tried downloading this. no luck so far…

can give more info please some screenshots? it does not pick up the keyboard or effects not working? I have v4 75% working on my using Sonoma 14.3.

**Comment by il-source** [comment]
> > does this work for anyone else? i have the white blackwidow v4 75% and tried downloading this. no luck so far…
> 
> can give more info please some screenshots? it does not pick up the keyboard or effects not working? I have v4 75% working on my using Sonoma 14.3.

it does not pick up the keyboard. i am using ventura 13.3.1

**Comment by il-source** [comment]
sorry, i am new to this, but is there anything else that i need to do other than downloading the release link?

**Comment by il-source** [comment]
i have updated my mac to sonoma 14.3 and i still have no luck with the device showing up.

**Comment by stickoking** [comment]
> sorry, i am new to this, but is there anything else that i need to do other than downloading the release link?

Just download the dmg file and install the app (drag app to applications folder), are you able to open the app ? It looks like this
![Screenshot 2024-02-05 at 3 13 48 PM](https://github.com/1kc/razer-macos/assets/41362698/84ac2663-398a-4055-befc-df6d113755c9)
is

**Comment by il-source** [comment]
> > sorry, i am new to this, but is there anything else that i need to do other than downloading the release link?
> 
> Just download the dmg file and install the app (drag app to applications folder), are you able to open the app ? It looks like this ![Screenshot 2024-02-05 at 3 13 48 PM](https://private-user-images.githubusercontent.com/41362698/302204160-84ac2663-398a-4055-befc-df6d113755c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxMTc5MDQsIm5iZiI6MTcwNzExNzYwNCwicGF0aCI6Ii80MTM2MjY5OC8zMDIyMDQxNjAtODRhYzI2NjMtMzk4YS00MDU1LWJlZmMtZGY2ZDExMzc1NWM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA1VDA3MjAwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0OTY2ODNmNmFlNGNmMTQ1YTg5OGU1N2JkMjM0Y2RkNzk5OWUwNjVlMTBhNDJjOTE0YTMwNzRlOGE1MTI0MjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vNjjn-yCLOxxYbSb94fyjhCIBVTZtbS60hSvIlA-jzQ) is

yes, i followed all those steps, but no device is being detected. i’ve tried different USB adapters and still no luck unfortunately.

**Comment by yehrhardt** [comment]
Does this build support the Blackshark V2 Pro (2023) by any chance?

**Comment by stickoking** [comment]
> > > sorry, i am new to this, but is there anything else that i need to do other than downloading the release link?
> > 
> > 
> > Just download the dmg file and install the app (drag app to applications folder), are you able to open the app ? It looks like this ![Screenshot 2024-02-05 at 3 13 48 PM](https://private-user-images.githubusercontent.com/41362698/302204160-84ac2663-398a-4055-befc-df6d113755c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxMTc5MDQsIm5iZiI6MTcwNzExNzYwNCwicGF0aCI6Ii80MTM2MjY5OC8zMDIyMDQxNjAtODRhYzI2NjMtMzk4YS00MDU1LWJlZmMtZGY2ZDExMzc1NWM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA1VDA3MjAwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0OTY2ODNmNmFlNGNmMTQ1YTg5OGU1N2JkMjM0Y2RkNzk5OWUwNjVlMTBhNDJjOTE0YTMwNzRlOGE1MTI0MjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vNjjn-yCLOxxYbSb94fyjhCIBVTZtbS60hSvIlA-jzQ) is
> 
> yes, i followed all those steps, but no device is being detected. i’ve tried different USB adapters and still no luck unfortunately.

@il-source  do you mean your laptop does not detect the keyboard and you're not able to use it? or you can use it but razer-macos does not detect it? If its just razer-macos app that can't detect it then can you go to system preference, click on general then click on about then at the bottom click on system report, and find the keyboard and send a screenshot also could you please open this issue on https://github.com/stickoking/razer-macos instead of this thread please. Sample of the screenshot is here 
![Screenshot 2024-02-05 at 4 26 15 PM](https://github.com/1kc/razer-macos/assets/41362698/93926b44-010c-4a00-814d-3f80a8d32862)

**Comment by rustanacexd** [comment]
works for me thanks a lot . now add cobra pro :D

**Comment by stickoking** [comment]
@rustanacexd can you raise a issue on https://github.com/stickoking/razer-macos for cobra pro?

**Comment by mhiecaaa** [comment]
i used your v11 but still Razer Kraken Kitty recognized by my mac but not detected on the razer macos. help :( 
![Screen Shot 2024-02-14 at 5 42 15 PM](https://github.com/1kc/razer-macos/assets/160010936/9e370523-3def-4849-9380-addef5e57a52)
